.github/workflows/publish.yml

```diff
@@ -1,10 +1,7 @@
 name: publish
```

Comment:
also, while you're here, could you make this `Publish`. Not sure what changed, but this name isn't being rendered properly like the other workflows in the Actions tab. the only difference being the capitalization